webapp/src/main/java/ar/edu/itba/paw/webapp/config/WebAuthConfig.java

```diff
@@ -52,48 +54,53 @@
 	@Override
 	protected void configure(final HttpSecurity http) throws Exception {
 		http
-				.userDetailsService(userDetailsService)
-				.sessionManagement()
-				.invalidSessionUrl("/login")
+						.sessionManagement()
+							.sessionCreationPolicy(SessionCreationPolicy.STATELESS)
+							.sessionFixation().migrateSession() // TODO: Should this be put to none since the sessionCreationPolicy is STATELESS ?
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1135)